theano/gpuarray/tests/test_dnn.py

```diff
@@ -26,6 +26,11 @@
 
 from theano.configdefaults import SUPPORTED_DNN_CONV_ALGO_FWD
 
+mode_with_gpu = mode_with_gpu.including()
+mode_without_gpu = mode_without_gpu.including()
+mode_with_gpu.check_py_code = False
+mode_without_gpu.check_py_code = False
```

Comment:
If you want to change mode_without_gpu globally to not check python code, that would be ok.